grouper/repositories/permission_grant.py

```diff
@@ -54,7 +61,38 @@ def grant_permission_to_service_account(self, permission, argument, service):
 
     def group_grants_for_permission(self, name, include_disabled_groups=False, argument=None):
         # type: (str, bool, Optional[str]) -> List[GroupPermissionGrant]
-        return self.repository.group_grants_for_permission(name, include_disabled_groups, argument)
+        grants = self.repository.group_grants_for_permission(
+            name, include_disabled_groups, argument
+        )
+        return grants
+
+    def group_paginated_grants_for_permission(
```

Comment:
Nit: can we add an optional pagination argument to `group_grants_for_permission` instead of making a separate function? Feels a bit cleaner, but feel free to disagree
(And same for the equivalent service account functions below, and for the equivalent functions in the service/usecase layers instead of the repository layer)

Reply:
Having an optional arg would make the return value a `Union`, which makes it a little annoying by introducing more casts